resources/testdata/glyphs2/WghtVar_ImplicitAxes.glyphs

```diff
@@ -20,6 +20,8 @@ capHeight = 0;
 descender = -200;
 id = m01;
 weightValue = 400;
+widthValue = 100;
+customValue = 0;
```

Comment:
did you edit these `widthValue` and `customValue` manually? I believe Glyphs doesn't write these fields out when they equal to their default (like many other attributes).

see:

https://github.com/googlefonts/glyphsLib/blob/main/Lib/glyphsLib/classes.py#L1563
https://github.com/googlefonts/glyphsLib/blob/f6e9c4a29ce764d34c309caef5118c48c156be36/Lib/glyphsLib/classes.py#L1517

Reply:
for v2, the default value of weightValue and widthValue is 100, the default value of customValue is 0

Reply:
Oh yes I did. OK, I'll take them out and read-or-default.

Reply:
Removed